char_gen.go

```diff
@@ -89,8 +89,8 @@ var charTypeNamesByFlag = map[CTFlag]string{
 
 // Re-generation trials for meeting requirements
 var (
-	MaxTrials   = 200              // MaxTrials how many time we try to generate before giving up
-	MinFailRate = 1.0 / 1000000000 // Minimum acceptable failure rate after MaxTrials
+	MaxTrials   = 200              // How many times we will try to generate before giving up
+	MaxFailRate = 1.0 / 1000000000 // Maximum acceptable failure rate after MaxTrials
```

Comment:
Ha. I think at one point I was thinking in terms of SuccessRate. Or maybe I just don't comprehend the difference between minima and maxima.